lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -129,6 +129,7 @@ def purchase(amount, payment_method, options={})
         add_email(post, options)
         add_3d_secure(post, options)
         add_echo(post, options)
+        post[:a9] = options[:transaction_type] if options[:transaction_type]
```

Comment:
I know it's one line, but you repeat this exact line three times, which is three changes to have a typo, and there's already precedents of `add_*`s in this gateway to ameliorate this exact issue (see `add_echo` or `add_email`). I'd thus really prefer this being `add_a9` or similar.